frontend/src/features/contacts/contact/detail/Organization.tsx

```diff
@@ -0,0 +1,215 @@
+          </Col>
+          <Col md="auto" className="ml-auto">
+            <Styled.StatusIndicators className={organization.isDisabled ? 'inactive' : 'active'}>
+              <FaCircle size={7} className="mr-2" />
+              <span data-testid="contact-organization-status">
+                {organization.isDisabled ? 'INACTIVE' : 'ACTIVE'}
+              </span>
+            </Styled.StatusIndicators>
+          </Col>
+        </Styled.RowAligned>
+        <Styled.RowAligned>
+          <Col className="ml-5">
+            <div>
+              <strong>Alias:</strong>
+            </div>
+            <div>
+              <span data-testid="contact-organization-alias">{organization.alias}</span>
+            </div>
+            <div>
+              <strong>Incorportation Number:</strong>
+            </div>
+            <div>
+              <span data-testid="contact-organization-incorporationNumber">
+                {organization.incorporationNumber}
+              </span>
+            </div>
+          </Col>
+        </Styled.RowAligned>
+      </FormSection>
+      <FormSection key={'contact-org-' + organization.id + '-contacts'} className="mb-4">
+        <Styled.H2Primary>Contact info</Styled.H2Primary>
+        <Styled.RowAligned className="pb-4">
+          <Col md={2}>
+            <strong>Email:</strong>
+          </Col>
+          <Col>
+            {personEmails.length === 0 && <span>N.A</span>}
+            {personEmails.map((field: ContactInfoField) => (
+              <Styled.RowAligned>
+                <Col md="4" data-testid="email-value">
+                  {field.info}
+                </Col>
+                <Col md="auto">
+                  <em>{field.label}</em>
+                </Col>
+              </Styled.RowAligned>
+            ))}
+          </Col>
+        </Styled.RowAligned>
+        <Styled.RowAligned>
+          <Col md={2}>
+            <strong>Phone:</strong>
+          </Col>
+          <Col>
+            {personPhoneNumbers.length === 0 && <span>N.A</span>}
+            {personPhoneNumbers.map((field: ContactInfoField, index: number) => (
+              <Styled.RowAligned>
+                <Col md="4" data-testid="phone-value">
+                  {phoneFormatter(field.info)}
+                </Col>
+                <Col md="auto">
+                  <em>{field.label}</em>
+                </Col>
+              </Styled.RowAligned>
+            ))}
+          </Col>
+        </Styled.RowAligned>
+      </FormSection>
+      <FormSection key={'contact-org-' + organization.id + '-address'} className="mb-4">
+        <Styled.H2Primary>Address</Styled.H2Primary>
+        {personAddresses.map((field: AddressField, index: number) => (
+          <Styled.RowAligned className="pb-3">
+            <Col md="4">
+              <strong>{field.label}:</strong>
+            </Col>
+            <Col data-testid="contact-organization-address">
+              <div>{field.address.streetAddress1} </div>
```

Comment:
I'm still a bit considered about the potential for NPEs here, but field.address should always be non-null I guess?

Reply:
fair point, all street addresses could be null